public/docs/ts/latest/guide/router.jade

```diff
@@ -1556,13 +1554,14 @@ h3#import-crisis-module Import crisis center module into the AppModule routes
   
   The `app-routing.module.ts` file retains the top-level application routes such as the default and wildcard routes.
 
-+makeExcerpt('app/app-routing.module.3.ts (v3)', 'v3')
++makeExcerpt('app/app-routing.module.3.ts (v3)')
 
 .l-main-section
-h2#relative-navigation Relative Navigation
 :marked
-  While building out the *Crisis Center* feature, you navigated to the
-  *Crisis Detail* route using a so-called **absolute path** that begins with a _slash_.
+  ### Relative navigation
```

Comment:
I always add an explicit anchor in case the section title changes; don't want to break external links.